projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php

```diff
@@ -292,6 +292,10 @@ private function build_task( $task ) {
 		$built_task['badge_text']   = $this->load_value_from_callback( $task, 'badge_text_callback' );
 		$built_task['isLaunchTask'] = isset( $task['isLaunchTask'] ) ? $task['isLaunchTask'] : false;
 
+		$repetition_count                 = $this->load_repetition_count( $task );
+		$built_task['repetition_count']   = $repetition_count ? $repetition_count : null;
+		$built_task['minimum_repetition'] = isset( $task['minimum_repetition_to_complete'] ) ? $task['minimum_repetition_to_complete'] : null;
```

Comment:
Minor naming thought: 'minimum_repetition' doesn't clearly communicate that it's a total, target, or maximum count. Might `target_repetitions` be more accurate? We could use that for both properties.
```suggestion
		$built_task['target_repetitions'] = isset( $task['target_repetitions'] ) ? $task['target_repetitions'] : null;
```